presto-kudu/src/main/java/io/prestosql/plugin/kudu/KuduClientConfig.java

```diff
@@ -151,4 +155,18 @@ public boolean isGroupedExecutionEnabled()
     {
         return groupedExecutionEnabled;
     }
+
+    @NotNull
+    public Duration getDynamicFilteringWaitTimeout()
+    {
+        return dynamicFilteringWaitTimeout;
+    }
+
+    @Config("kudu.dynamic-filtering-wait-timeout")
```

Comment:
Let's make it `kudu.dynamic-filtering.wait-timeout`

Reply:
Modified